backend/pkg/api/router.go

```diff
@@ -86,9 +85,13 @@ func addRoutes(hs *handlers.HandlerService, publicRouter, internalRouter *mux.Ro
 		{http.MethodGet, "/healthz-loadbalancer", hs.PublicGetHealthzLoadbalancer, nil},
 
 		{http.MethodPost, "/login", nil, hs.InternalPostLogin},
-		{http.MethodPost, "/logout", nil, hs.InternalPostLogout},
 
-		{http.MethodPost, "/oauth/token", hs.PublicPostOauthToken, nil},
+		{http.MethodGet, "/mobile/authorize", nil, hs.InternalPostAuthorize},
+		{http.MethodPost, "/mobile/equivalent-exchange", nil, hs.InternalExchangeLegacyMobileAuth},
+		{http.MethodPost, "/mobile/notify/register", nil, hs.InternalRegisterMobilePushToken},
```

Comment:
If we want this endpoint to fit with the other notification endpoints, we could have it be:
```
PUT /api/i/users/me/notifications/settings/paired-devices/{client_id}/token
```